packages/react-data-grid/style/rdg-core.less

```diff
@@ -29,4 +29,5 @@
 .rdg-grid {
   // min-height is here to show the horizontal scrollbar when there are no rows
   min-height: 1px;
+  margin-bottom: 8px;
```

Comment:
Add small space between the bottom row and the bottom border, as the editable small square will increase the height size a little bit, so it's better to leave the margin in advance to avoid some randomly bug that a cell cannot be edited.

![image](https://user-images.githubusercontent.com/25043398/68976435-613eb600-07bb-11ea-9474-fc4295042527.png)